File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.py
import sys
input = sys.stdin.readline

C = int(input())
for _ in range(C):
    temp = list(map(int,input().split()))
    total = 0
    for i in range(1,len(temp)):
        total = total + temp[i]

    avg = total // temp[0]
    over = 0

    for i in range(1,len(temp)):
        if temp[i] > avg:
            over = over + 1

    print("%.3f" %(over / temp[0] * 100) +"%")

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.js
let [n,...arr] = require('fs').readFileSync('dev/stdin').toString().trim().split('\n')

for(let i =0; i<n; i++){
  let temp = arr[i].split(' ').map(a=>Number(a))
  let count = temp.shift()
  let sum = temp.reduce((a,b)=>a+b,0)
  let avg = sum/count
  let ratio = 0
  for(let i=0; i<count; i++){
    if (temp[i]>avg){
      ratio+=1
    }
  }
  console.log(((ratio/count)*100).toFixed(3)+'%')
}